### ISSUE TITLE
Repo Status

### ISSUE THREAD

**Issue by aersam** [tiberius-rs/tiberius#321]
Is this project still alive?

Last commit is some time ago and there seem to be lot's of open issues and open PR's

**Comment by janpio** [comment]
It is alive and used in Prisma. 

But we do not need many changes right now ourselves, so it is a bit dormant.
If you are using Tiberius, it would be nice if you could help out with responding to issue and/or PRs, with reviews and comments.

**Comment by aersam** [comment]
Great to hear that!

I am using mostly the Bulk Insert Part of Tiberius and also did a PR concerning that: https://github.com/prisma/tiberius/pull/312 

In addition I'm testing #315 , if this one works it would be great to get merged.

So for bulk Insert stuff I could surely help testing, I use it to load GB's of data every day

**Comment by aersam** [comment]
#315 works or at least does not cause errors :) I however still have some issues with large text fields, I'll open a separate issue for that and hope to find some solution.

In generall I'm not too deep into TDS itself, I know MS SQL Server by heart, however

**Comment by lpj145** [comment]
Is someone alive here? tiberius now is used for a lot of people that was dealing with sql server but a lot of issues and pr's still open... @janpio @esheppa

**Comment by janpio** [comment]
Yes, we are alive. Pinging random people will not make anyone happy though.

Prisma is currently happy with the state of this crate. If you need any functionality from open PRs, I suggest you work with a fork until we have the time to review these and merge them.

**Comment by lpj145** [comment]
@janpio Have you had a error like "perhaps two different versions of crate?" yeah, is very difficult to use your own version of this crate.

**Comment by max-sixty** [comment]
Would it be possible to create a release with recent updates? The most recent release was ~a year ago. Thank you!

**Comment by xxchan** [comment]
> Would it be possible to create a release with recent updates?

+1. Specifically want to have #285 published, which can solve an audit alarm @janpio

**Comment by MattJackson** [comment]
The project is active again — it's been moved to the `tiberius-rs` org with new maintainers, dependencies (rustls 0.23) and the panic/DoS fixes are landing, and releases are being cut. Closing this status thread; see #427 for the ongoing maintenance/governance discussion. Thanks for your patience!
